Modules/Quotas/test/Quotas.Application.Tests/Tests/IntegrationEvents/TierDeleted/TierOfIdentityChangedIntegrationEventHandlerTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿using Backbone.Modules.Quotas.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Quotas.Application.IntegrationEvents.Incoming.TierOfIdentityChanged;
+using Backbone.Modules.Quotas.Application.Metrics;
+using Backbone.Modules.Quotas.Domain.Aggregates.Identities;
+using Backbone.Modules.Quotas.Domain.Aggregates.Metrics;
+using Backbone.Modules.Quotas.Domain.Aggregates.Tiers;
+using Enmeshed.UnitTestTools.Data;
+using FakeItEasy;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Quotas.Application.Tests.Tests.IntegrationEvents.TierDeleted;
+public class TierOfIdentityChangedIntegrationEventHandlerTests
+{
+    [Fact]
+    public async Task Handler_updates_identity_with_new_tier()
```

Comment:
These are Handler specific tests. So who should update the identity if not the Handler?
```suggestion
    public async Task Updates_identity_with_new_tier()
```